main.py

```diff
@@ -132,10 +44,9 @@ def new_task():
         print("Nie podano poprawnej nazwy zadania!")
         new_task()
     else:
```

Comment:
W końcu rozkminiłem jak się pozbyć tego brzydkiego wywoływania funkcji ponownie, gdy nie spełniasz warunku.
```python
def new_task():
    while True:
        task_name = input("Podaj nazwę nowego zadania: ")
        if len(task_name) > 0:
            break
        print("Nie podano poprawnej nazwy zadania!")

    new = Task(task_name, False)
```

Reply:
o Boże w task_edition mam tak samo, to jest brzydkie?

Reply:
Wyeliminowano brzydkie wywołanie funkcji ponownie new_task i task_edition pętlą while.